packages/dd-trace/index.js

```diff
@@ -1,7 +1,11 @@
 'use strict'
 
-const TracerProxy = require('./src/proxy')
+if (!global.ddtrace) {
```

Comment:
Adding a ddtrace global is bound to break some folks' test suites. Are we sure we need to do this in this way?

Reply:
It's the only way to work around Next, or more specifically its use of Webpack, without ending up with multiple instances. I could add additional checks if we don't want this to _always_ happen. However, it's worth noting that generally speaking, it's not supported to have multiple tracers initialized within a process and it could cause various stability issues, so it's probably safer in any case to ensure there can only be one regardless of how dependencies are managed.